lib/manager/npm/post-update/index.ts

```diff
@@ -541,7 +541,7 @@ export async function getAdditionalFiles(
       (upgrade) => upgrade.npmLock === npmLock
     );
     const res = await npm.generateLockFile(
-      fullLockFileDir,
+      lockFileDir,
       env,
       fileName,
```

Comment:
We should use consistent making of this `lockFileDir` and `fileName`. Within the function this calls, we use `cwd` and `filename` with lower case `n`. Please unify them and then we decide if we can simplify further

Reply:
@rarkins do you prefer me to use `lockFileDir` and `fileName` or `cwd` and `filename`? For now it is not clear which one you prefer.

Reply:
I think `lockFileDir` and `fileName`. We use `fileName` about 5x more often than `filename` in our repo.

Reply:
agreed